M2/Macaulay2/packages/RationalMaps.m2

```diff
@@ -3,7 +3,7 @@ Version => "0.3", Date => "August 11th, 2019", Authors => {
      {Name => "Karl Schwede",
      Email=> "kschwede@gmail.com",
      HomePage=> "http://www.math.utah.edu/~schwede"
-     }, --Karl Schwede was partially supported by  NSF FRG Grant DMS #1265261/1501115, NSF CAREER Grant DMS #1252860/1501102
+     }, --Karl Schwede was partially supported by  NSF FRG Grant DMS #1265261/1501115, NSF CAREER Grant DMS #1252860/1501102, and NSF grant #1801849
```

Comment:
You can now use the `Acknowledgment` keyword in SimpleDoc for this purpose!

Reply:
Well, it'll be in v1.17, but it is in the development branch now.

Reply:
I'll wait then for now.